stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/utils/LiveDataTestUtil.kt

```diff
@@ -13,24 +11,12 @@ internal suspend fun waitForSetUser(
     client: ChatClient,
     user: User,
     token: String,
-    timeMillis: Long = 5000
+    timeMillis: Long = 5000,
 ) {
     val lock = CompletableDeferred<Unit>()
-    client.setUser(
-        user,
-        token,
-        object : InitConnectionListener() {
-            override fun onError(error: ChatError) {
-                super.onError(error)
-                lock.complete(Unit)
-            }
-
-            override fun onSuccess(data: ConnectionData) {
-                super.onSuccess(data)
-                lock.complete(Unit)
-            }
-        }
-    )
+    client.connectUser(user, token).enqueue {
+        lock.complete(Unit)
```

Comment:
Now that we have a `Call` here, we don't need the "lock" anymore.
The `delay()` and `withTimeout()` maybe is not needed anymore neither

Reply:
You will need to use `execute()` to block the call, instead of `enqueue()` if we want to remove the old implementation

Reply:
Try removing them and not breaking tests, I spent hours on it and couldn't get it simpler than this 😭 

Robolectric is a pain